vsintegration/tests/unittests/VisualFSharp.UnitTests.fsproj

```diff
@@ -38,26 +38,26 @@
     <Compile Include="Tests.TaskReporter.fs" />
     <Compile Include="Tests.Watson.fs" />
     <Compile Include="Tests.XmlDocComments.fs" />
-    <Compile Include="Tests.LanguageService.General.fs" />
-    <Compile Include="Tests.LanguageService.Completion.fs" />
-    <Compile Include="Tests.LanguageService.IncrementalBuild.fs" />
-    <Compile Include="Tests.LanguageService.GotoDefinition.fs" />
-    <Compile Include="Tests.LanguageService.NavigationBar.fs" />
-    <Compile Include="Tests.LanguageService.ParameterInfo.fs" />
-    <Compile Include="Tests.LanguageService.QuickInfo.fs" />
-    <Compile Include="Tests.LanguageService.QuickParse.fs" />
-    <Compile Include="Tests.LanguageService.Script.fs" />
-    <Compile Include="Tests.LanguageService.TimeStamp.fs" />
-    <Compile Include="Tests.LanguageService.ErrorList.fs" />
-    <Compile Include="Tests.LanguageService.ErrorRecovery.fs" />
-    <Compile Include="Tests.ProjectSystem.Configs.fs" />
-    <Compile Include="Tests.ProjectSystem.Miscellaneous.fs" />
-    <Compile Include="Tests.ProjectSystem.MultiTargeting.fs" />
-    <Compile Include="Tests.ProjectSystem.ProjectItems.fs" />
-    <Compile Include="Tests.ProjectSystem.Project.fs" />
-    <Compile Include="Tests.ProjectSystem.References.fs" />
-    <Compile Include="Tests.ProjectSystem.RoundTrip.fs" />
-    <Compile Include="Tests.ProjectSystem.UpToDate.fs" />
+    <Compile Include="LanguageService\Tests.LanguageService.General.fs" />
```

Comment:
Could this get renamed to `LegacyLanguageService`?

Reply:
sure thing 👍

Reply:
![](http://i.ytimg.com/vi/hz5jZeNdfcY/hqdefault.jpg)